### ISSUE TITLE
How did you come up with such a large number of filters?

### ISSUE THREAD

**Issue by rohitgarud** [camaradesuk/ASySD#15]
Sorry..This is not an issue, but as there is no discussion section, so asking it here. Did you follow some iterative method to add filters sequentially or use some kind of heuristic for it?

**Comment by kaitlynhair** [comment]
Hi - sorry I didn't reply sooner. I'm just tidying up a few of the outstanding issues now.  
I developed ASySD in response to a specific problem I had - I couldn't dedup huge numbers effectively in Endnote and couldn't find another solution at the time. I used several datasets we had within our research group from past systematic reviews. I basically added filters I thought made sense and then did A LOT(!) of trial and error and manually checking false positives / false negatives on those testing datasets and tweaking the filters continually. When I was happy with it, I tested the final version on 5 unseen (but similar) datasets and it worked pretty well. 

I really didn't follow a specific process back then as it was very experimental at the start. In future I plan to re-evaluate the match filters using a more systematic approach and adapt them to suit different article types / research databases.
